hvplot/__init__.py

```diff
@@ -209,6 +209,54 @@ def _hook_patch_docstrings(backend):
 
 @_wraps(_pn.bind)
 def bind(function, *args, **kwargs):
+    """
+    Returns a *reactive* function that *depends* on widgets, parameters or Python objects.
+    This means that the "reactive" function can be automatically invoked whenever the underlying
+    widget or parameter values change. The widgets can be Panel or IpyWidgets.
+    
+    `hv.bind` can be **used to start your `.interactive` pipeline** by loading data depending
+    on widgets or pameters values.
+    
+    Reference: https://github.com/holoviz/hvplot/issues/825
+
+    :Example:
+
+    >>> import pandas as pd
+    >>> import panel as pn
+
+    >>> import hvplot
+
+    >>> pn.extension()
+
+    >>> experiment = pn.widgets.IntSlider(value=5, start=0, end=10, step=1, name="Experiment")
```

Comment:
Depending on the function used as input, we'll have to make sure the right widget is used. In this case I'd say a `Select` would have been more appropriate.

Reply:
Thanks. I did not think it mattered. But I think you are right.

Reply:
As experiment was changed to alpha, I kept the IntSlider for alpha. But changed the number of rows select to a RadioButtonGroup. That makes sense to me.